admin-dev/themes/new-theme/js/components/tinymce-editor.js

```diff
@@ -0,0 +1,236 @@
+
+      if (translatedField.length && tabContainer.length) {
+        const textareaLocale = translatedField.data('locale');
+        const textareaLinkSelector = '.nav-item a[data-locale="'+textareaLocale+'"]';
+
+        $(textareaLinkSelector, tabContainer).on('shown.bs.tab', () => {
+          const editor = tinyMCE.get(textarea.id);
+          if (editor) {
+            //Reset content to force refresh of editor
+            editor.setContent(editor.getContent());
+          }
+        });
+      }
+    });
+  }
+
+  /**
+   * Loads the TinyMCE javascript library and then init the editors
+   *
+   * @param config
+   */
+  loadAndInitTinyMCE(config) {
+    if (this.tinyMCELoaded) {
+      return;
+    }
+
+    this.tinyMCELoaded = true;
+    var path_array = config.baseAdminUrl.split('/');
+    path_array.splice((path_array.length - 2), 2);
+    var final_path = path_array.join('/');
+    window.tinyMCEPreInit = {};
+    window.tinyMCEPreInit.base = final_path+'/js/tiny_mce';
+    window.tinyMCEPreInit.suffix = '.min';
+    $.getScript(final_path+'/js/tiny_mce/tinymce.min.js', () => {this.setupTinyMCE(config)});
+  }
+
+  /**
+   * Replace initial TinyMCE icons with material icons
+   */
+  changeToMaterial() {
+    let materialIconAssoc = {
+      'mce-i-code': '<i class="material-icons">code</i>',
+      'mce-i-none': '<i class="material-icons">format_color_text</i>',
+      'mce-i-bold': '<i class="material-icons">format_bold</i>',
+      'mce-i-italic': '<i class="material-icons">format_italic</i>',
+      'mce-i-underline': '<i class="material-icons">format_underlined</i>',
+      'mce-i-strikethrough': '<i class="material-icons">format_strikethrough</i>',
+      'mce-i-blockquote': '<i class="material-icons">format_quote</i>',
+      'mce-i-link': '<i class="material-icons">link</i>',
+      'mce-i-alignleft': '<i class="material-icons">format_align_left</i>',
+      'mce-i-aligncenter': '<i class="material-icons">format_align_center</i>',
+      'mce-i-alignright': '<i class="material-icons">format_align_right</i>',
+      'mce-i-alignjustify': '<i class="material-icons">format_align_justify</i>',
+      'mce-i-bullist': '<i class="material-icons">format_list_bulleted</i>',
+      'mce-i-numlist': '<i class="material-icons">format_list_numbered</i>',
+      'mce-i-image': '<i class="material-icons">image</i>',
+      'mce-i-table': '<i class="material-icons">grid_on</i>',
+      'mce-i-media': '<i class="material-icons">video_library</i>',
+      'mce-i-browse': '<i class="material-icons">attachment</i>',
+      'mce-i-checkbox': '<i class="mce-ico mce-i-checkbox"></i>',
+    };
+
+    $.each(materialIconAssoc, function (index, value) {
+      $('.' + index).replaceWith(value);
+    });
+  }
+
+  /**
+   * Updates the characters counter
+   *
+   * @param id
+   */
+  handleCounterTiny(id) {
+    let textarea = $('#'+id);
+    let counter = textarea.attr('counter');
```

Comment:
```suggestion
    const counter = textarea.attr('counter');
```

Reply:
use data attribute instead?

Reply:
I prefer to keep it that way for now, because it would require to also update (and complexify) Symfony forms (attributes are defined in them) and twig form templates